core/src/test/java/org/mskcc/oncotree/utils/OncotreeUtilTestConfig.java

```diff
@@ -0,0 +1,826 @@
+/** Copyright (c) 2017 Memorial Sloan-Kettering Cancer Center.
+ *
+ * This library is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY, WITHOUT EVEN THE IMPLIED WARRANTY OF
+ * MERCHANTABILITY OR FITNESS FOR A PARTICULAR PURPOSE.  The software and
+ * documentation provided hereunder is on an "as is" basis, and
+ * Memorial Sloan-Kettering Cancer Center
+ * has no obligations to provide maintenance, support,
+ * updates, enhancements or modifications.  In no event shall
+ * Memorial Sloan-Kettering Cancer Center
+ * be liable to any party for direct, indirect, special,
+ * incidental or consequential damages, including lost profits, arising
+ * out of the use of this software and its documentation, even if
+ * Memorial Sloan-Kettering Cancer Center
+ * has been advised of the possibility of such damage.
+*/
+
+package org.mskcc.oncotree.utils;
+
+import java.util.*;
+import static org.junit.Assert.fail;
+import org.mockito.Mockito;
+import org.mskcc.oncotree.model.MainType;
+import org.mskcc.oncotree.model.TumorType;
+import org.mskcc.oncotree.model.Version;
+import org.mskcc.oncotree.topbraid.OncoTreeNode;
+import org.mskcc.oncotree.topbraid.OncoTreeRepository;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import static org.junit.Assert.fail;
+
+/**
+ *
+ * @author heinsz
+ */
+@Configuration
+@ComponentScan(basePackages = {"org.mskcc.oncotree.utils","org.mskcc.oncotree.topbraid"})
+public class OncotreeUtilTestConfig {
+    
+    @Bean
+    public ArrayList<OncoTreeNode> oncoTreeRepositoryMockResponse() throws Exception {
+        return setupOncotreeRepositoryMockResponse();
+    }
+    
+    @Bean
+    public Map<String, TumorType> expectedTumorTypeMap() throws Exception {
+        return setupExpectedTumorTypeMap();
+    }
+    
+    @Bean
+    public List<MainType> expectedMainTypeList() throws Exception {
+        return setupExpectedMainTypeList();
+    }
+    
+    @Bean
+    public Version mockVersion() {
+        return setupMockVersion();
+    }
+    
+    @Bean
+    public OncoTreeRepository oncoTreeRepository() {
+        return Mockito.mock(OncoTreeRepository.class);
+    }
+    
+    @Bean
+    public TumorTypesUtil tumorTypesUtil() {
+        return new TumorTypesUtil();
+    }
+    
+    @Bean
+    public MainTypesUtil mainTypesUtil() {
+        return new MainTypesUtil();
+    }
+    
+    private Version setupMockVersion() {
```

Comment:
note to self: we may need more of these later for testing variation across different versions